tilequeue/query.py

```diff
@@ -64,21 +63,26 @@ def jinja_filter_bbox(bounds, srid=900913):
     return bbox
 
 
-def build_feature_queries(bounds, layer_data, zoom):
+def build_feature_queries(unpadded_bounds, layer_data, zoom):
     queries_to_execute = []
     for layer_datum in layer_data:
+        query_bounds_pad_fn = layer_datum['query_bounds_pad_fn']
+        if query_bounds_pad_fn:
+            padded_bounds = query_bounds_pad_fn(unpadded_bounds)
+        else:
+            padded_bounds = unpadded_bounds
```

Comment:
Don't think so, as in I think functions are always truthy. I could alternatively make it return none to represent no buffer operation, but this seemed easier at the time. Is that what you had in mind?